docs/usermanual.md

```diff
@@ -116,10 +116,36 @@ tools available online like [this one](https://string-functions.com/string-hex.a
 
 It is possible report as a measure a NGSI-v2 or NGSI-LD payload when related device/group is configured with
 `payloadType` `ngsiv2` or `ngsild`. In these cases payload is ingested as measure where entity attributes are measure
-attributes and `id` and `type` are ignored, since `id` and `type` from device/group configuration provisioned are used,
-as well as `actionType`.
+attributes. The entity `id` and `type` are also include as attributes `ngsi_id` and `ngsi_type` (both with attribute
```

Comment:
This case (id is included as ngsi_id and type is included as ngsi_type) is just for ngsiv2 or ngsild payload. Not for a tipical iotagent measure

Reply:
This is under NGSI-v2 and NGSI-LD Measure reporting section

Reply:
Following this PR https://github.com/telefonicaid/iotagent-json/pull/807 final suffix will be "measure_" for all cases

Reply:
Changed here https://github.com/telefonicaid/iotagent-json/pull/806/commits/a9485d45118a668dc14ab84ee22bb01de1126914